Integrations/integration-GoogleResourceManager.yml

```diff
@@ -0,0 +1,1020 @@
+category: IT Services
+commonfields: {id: Google Resource Manager, version: -1}
+configuration:
+- defaultvalue: ''
+  display: The Project ID of the project resource in which you created the Service
+    Account. The value can be found in the Service Account Key File which should have
+    downloaded when creating the Service Account in the Google Cloud Platform.
+  name: project_id
+  options: []
+  required: true
+  type: 0
+- defaultvalue: ''
+  display: Value of the 'private_key_id' field in your JSON Service Account Key File.
+  name: private_key_id
+  options: []
+  required: true
+  type: 4
+- defaultvalue: ''
+  display: Value of the 'private_key' field in your JSON Service Account Key File.
+  name: private_key
+  options: []
+  required: true
+  type: 4
+- defaultvalue: ''
+  display: Value of the 'client_email' field in your JSON Service Account Key File.
+  name: client_email
+  options: []
+  required: true
+  type: 0
+- defaultvalue: ''
+  display: Value of the 'client_id' field in your JSON Service Account Key File.
+  name: client_id
+  options: []
+  required: true
+  type: 4
+- defaultvalue: ''
+  display: Value of the 'client_x509_cert_url' field in your JSON Service Account
+    Key File.
+  name: client_x509_cert_url
+  options: []
+  required: true
+  type: 0
+description: Google Cloud Platform Resource Manager
+detaileddescription: Use these detailed instructions in order to retrieve the API
```

Comment:
Please edit this and put here instructions on how to configure the integration (i.e. where to get all the required parameters)